### ISSUE TITLE
SpaceAfterClosingBrace should not complain if closure is used as switch-case condition

### ISSUE THREAD

**Issue by Vampire** [CodeNarc/CodeNarc#452]
CodeNarc thinks this
```groovy
switch {
    case { this.hasProperty("$parameterName") }:
        break
}
```
should look like
```groovy
switch {
    case { this.hasProperty("$parameterName") } :
        break
}
```
But as it is quite uncommon to have spaces before the case colon, I'd like to ask for a change here, or at least for an option.

**Comment by chrismair** [comment]
Agreed.

**Comment by chrismair** [comment]
Available with CodeNarc 1.6.
